types/pollyjs__adapter-puppeteer/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for @pollyjs/adapter-puppeteer 4.0
+// Project: https://github.com/netflix/pollyjs/tree/master/packages/@pollyjs/adapter-puppeteer
+// Definitions by: offirgolan <https://github.com/offirgolan>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import Adapter from '@pollyjs/adapter';
+
+export default class PuppeteerAdapter extends Adapter {}
```

Comment:
```suggestion
class PuppeteerAdapter extends Adapter {}
export = PuppeteerAdapter;
```

If you look at their [runtime behavior](https://runkit.com/embed/597ogce4iypk) they transpile a `default` into a `module.exports` assignment, so the types need to reflect the runtime reality, not the original source.

Reply:
Updated. There are other polly packages that need this change. Should I address that in a follow up PR or include it here?